platform/src/main/python/dlpx/virtualization/platform/_linked.py

```diff
@@ -331,19 +360,16 @@ def _internal_staged_pre_snapshot(self, request):
         linked_source = request.staged_source.linked_source
```

Comment:
Not your change but did you also notice we are duplicating these line of codes at multiple places?

Reply:
Yes, but did not update it as it may cause regression and the same code is across the platform package.